### PR TITLE
Adds Description to serviced service runs

### PR DESCRIPTION
**Pull request by ed-**
Closes CC-1102

### CODE REVIEW

File: Products/ZenModel/migrate/addDescriptionToCommands.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        for service in ctx.services:
+            commands = service.commands
+            if not commands:
+                continue
+            for command in commands:
+                if hasattr(command, "Description"):
```
Comment (ed-) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: fixme

File: Products/ZenModel/migrate/addDescriptionToCommands.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                        continue
+                command.description = newDescriptions.get(command.name, "")
```
Comment (ed-) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: needs sdk

File: Products/ZenModel/migrate/addDescriptionToCommands.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    version = Migrate.Version(5, 1, 70)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        for service in ctx.services:
```
Comment (tgjonestx) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: I think we should be defensive here and only apply these changes to the products we specifically want to change. See lines 42-45 of fixZopeHealthCheck.py.

File: Products/ZenModel/migrate/addDescriptionToCommands.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            commands = service.commands
+            if not commands:
+                continue
+            for command in commands:
```
Comment (tgjonestx) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: Are we sure that all of the command names across all of the services are unique?  
Names like 'check' and 'initialize' sound kind of generic; i.e. 2 different services have different commands that happen to have the same name.

Comment (ed-) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: https://gist.github.com/ed-/4a2e6d22327fd6c62904

Pretty sure

File: Products/ZenModel/migrate/addDescriptionToCommands.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    "model-zenmigrate": "Import zenmigrate information for Import4.",
+    "model-zenpack": "Import Zenpack information for Import4.",
+    "perf-abort-cleanup": "Deprecated.",
+    "perf-import": "Import performance data for Import4.",
+    "perf-verify": "Verify performance data for Import4.",
+    "rebuild_zodb_session": "Drop and recreate the zodb_session database.",
+    "reportmail": "Generate and email a status report.",
+    "upgrade": "Start a service migration.",
+    "zendmd": "Start an interactive Zenoss DMD session.",
+    "zenmib": "Load, convert, or import MIB files into the Zenoss DMD.",
```
Comment (ed-) [inline Products/ZenModel/migrate/addDescriptionToCommands.py]: Nope, should have stayed "and"

**Comment by ed-** [comment]
retest this

**Comment by rDr4g0n** [comment]
retest this

**Comment by tgjonestx** [comment]
@ed- do the descriptions in this PR match those https://github.com/zenoss/zenoss-service/pull/232?

If not, the descriptions here should match those we end up with after https://github.com/zenoss/zenoss-service/pull/232 is merged

**Comment by ed-** [comment]
I should have those changes here momentarily.

**Pull request merged**